app/scripts/modules/core/src/instance/InstanceList.tsx

```diff
@@ -0,0 +1,15 @@
+export interface IInstanceListProps {
+  hasDiscovery: boolean;
+  hasLoadBalancers: boolean;
+  instances: any[];
+  sortFilter: any;
+  serverGroup: any;
+}
+
+export const instanceListBindings: Record<keyof IInstanceListProps, string> = {
```

Comment:
yes, that is a copy of the `scope:` object from the directive definition object.  I decided it was not a good use of my time to refactor the directives to components, and `angular2react` can handle either